framework/src/bcs/MatNeumannBC.C

```diff
@@ -0,0 +1,42 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "MatNeumannBC.h"
+
+registerMooseObject("MooseApp", MatNeumannBC);
+
+defineLegacyParams(MatNeumannBC);
+
+InputParameters
+MatNeumannBC::validParams()
+{
+  InputParameters params = IntegratedBC::validParams();
+  params.addParam<Real>("value", 0.0, "The value to be enforced on the boundary.");
```

Comment:
```suggestion
  params.addParam<Real>("value", 1.0, "The value to be enforced on the boundary.");
```

I suggest making the default here 1.0, as it will be very confusing for users who just specify a material property in the input to find the flux always set to zero, regardless of the value of the material property.

Reply:
Agreed, that makes sense.